Mathlib/CategoryTheory/Shift/Adjunction.lean

```diff
@@ -215,6 +215,47 @@ lemma mk' (h : NatTrans.CommShift adj.unit A) :
       Functor.commShiftIso_id_hom_app, whiskerRight_app, id_comp,
       Functor.commShiftIso_comp_hom_app] using congr_app (h.comm' a) X⟩
 
+variable [adj.CommShift A]
+
+lemma commShift_unit : NatTrans.CommShift adj.unit A := commShift_unit'
```

Comment:
This should be directly in the `NatTrans` namespace, which should make unnecessary the `'`.

Reply:
> I can put these lemmas in the `Adjunction` namespace (this is the new commit), but I don't understand the remark about the `'`.

The PR adds '`' to some lemmas, which seems unnecessary to me if the lemmas are put to the `Adjunction` namespace rather than `CommShift`.

Reply:
I still don't understand. The PR doesn't add `'` to lemmas, just to the fields of the structure `Adjunction.CommShift`, because I wanted to reserve the names without `'` for lemmas as is done, for example, with `NatTrans.CommShift`.

Reply:
Just do not add `'` when it is not necessary.

Reply:
I merely copied this model:
```
/-- If `τ : F₁ ⟶ F₂` is a natural transformation between two functors
which commute with a shift by an additive monoid `A`, this typeclass
asserts a compatibility of `τ` with these shifts. -/
class CommShift : Prop where
  comm' (a : A) : (F₁.commShiftIso a).hom ≫ whiskerRight τ _ =
    whiskerLeft _ τ ≫ (F₂.commShiftIso a).hom

namespace CommShift

section

variable {A}
variable [NatTrans.CommShift τ A]

lemma comm (a : A) : (F₁.commShiftIso a).hom ≫ whiskerRight τ _ =
    whiskerLeft _ τ ≫ (F₂.commShiftIso a).hom := by
  apply comm'
```
from `Mathlib.CategoryTheory.Shift.CommShift`, which you apparently wrote yourself.

Reply:
(But okay, I've removed the primes here...)